### PR TITLE
[aes] Clarify that the unit needs to be idle before initialization

### PR DESCRIPTION
**Pull request by vogelpi**

### CODE REVIEW

**Comment by vogelpi** [comment]
Thanks @silvestrst for your review.

> Maybe it is worth somewhere in "Datapath Architecture and Operation" to mention that IV has different meaning in CTR (Initial counter value) and CBC (Initialisation Vector) modes?

This is already stated in the spec. See Point 2 of this very section: "The processor provides the initialization vector (IV) or initial counter value to the four IV registers via TL-UL bus interface in CBC or CTR mode, respectively." Do you think this is sufficient? I can edit the spec in a new PR, or rephrase it to make it more clear. Just let me know.

**Comment by silvestrst** [comment]
> Do you think this is sufficient? I can edit the spec in a new PR, or rephrase it to make it more clear. Just let me know.

Yes that's good, thanks, apologies - I have missed it the first time :/
